tests/Avalonia.Controls.UnitTests/GridTests.cs

```diff
@@ -1382,6 +1376,245 @@ public void Correct_Grid_Bounds_When_Child_Control_Has_DesiredSize_Larger_Than_A
             Assert.Equal(new Size(100, 100), grid.Bounds.Size);
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void Change_Column_Width_Should_Invalidate_Grid(bool setUsingAvaloniaProperty)
```

Comment:
Nits:  `Change_Column_Width_Should_Invalidate_Grid` -> `Changing_Column_Width_Should_Invalidate_Grid` 

Other similar tests should have the same correction too